qiita_ware/test/test_ebi.py

```diff
@@ -405,6 +410,72 @@ def test_generate_curl_command(self):
                          '%20ebi_access_key%3D"')
         self.assertEqual(obs, exp_with_cert)
 
+    def test_generate_demultiplexed_fastq(self):
+        # generating demux file for testing
+        if not isdir(self.temp_dir):
+            makedirs(self.temp_dir)
```

Comment:
Can you add a `tearDown` function to the tests class and make sure that this directory gets removed after the execution of the test? Is fine for me if you want to add it to the todo list and do it later, but we should make sure that we  are green and we clean the environment :smile: